web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -24,25 +24,20 @@ import {
   CardContent,
   CardHeader,
   Chip,
-  Dialog,
```

Comment:
I factored out the confirmation dialog used in this component and changed it to use the new `ConfirmationDialog` component.